src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/SetNoteStatusRequestHandlerTests.cs

```diff
@@ -85,6 +85,56 @@ public async Task ExternalUser_CanSetApprovedStatus()
             note.Status.Should().Be(NoteStatus.Approved);
         }
 
+        [Fact]
+        public async Task ExternalUser_CanSetRejectedStatus()
```

Comment:
instead of copy-pasting the same test, I think re-use the 1 test, as passing Inline Data the statuses which you want to test for. So you could have 1 test method for Approve, Reject and Return statuses

Reply:
This is like a good idea - but in this particular case it won't work - please check email I have sent on the subject.

Reply:
Assert block resolved with this code:
note.Status.Should().Be(status.ToDomainEnumeration<NoteStatus>());